ansible/group_vars/tag_Environment_discovery

```diff
@@ -20,8 +20,10 @@ register_groups:
     - datatype
     - field
 
-  multi:
+  address:
     - address
```

Comment:
wait, why are you doing this? `address` only needs to be separate in `test`, right?